modules/nodes-cluster-resource-levels-job.adoc

```diff
@@ -53,6 +53,29 @@ EOF
 ----
 $ oc create sa cluster-capacity-sa
 ----
++
```

Comment:
This TIP isn't quite accurate.  There are at least two secrets that are added when the `oc adm` command is run.  There could be other resources that are added silently as well.  I think we need some SME advice to learn the declarative equivalent.